src/stories/containers/users/delete-account/delete-account.tsx

```diff
@@ -14,57 +14,83 @@ import { toast } from 'react-toastify';
 import Notification, { ContainerNotification } from '../../../components/notification/notification';
 import CheckMark from '../../../components/svg/check-mark';
 import Warning from '../../../components/svg/warning';
+import { LOGIN_REQUEST } from '../../auth/login/login.api';
+import request from 'graphql-request';
+import { GRAPHQL_ENDPOINT } from '../../../../config/endpoints';
 
 export default () => {
-  const testingPassword = '1234';
   const router = useRouter();
-  const { clientRequest } = useAuthContext();
+  const { clientRequest, user } = useAuthContext();
   const { userName, id } = router.query;
   const [value, setValue] = useState('');
   const { isLight } = useThemeContext();
   const handleChange = useCallback((value: React.ChangeEvent<HTMLInputElement>) => {
     setValue(value.target.value);
   }, []);
 
+  const handleGoBack = useCallback(() => {
+    router.back();
```

Comment:
same thing that the previous `goBack` functions

Reply:
You are right I will make the change